redisdb/tests/test_default.py

```diff
@@ -81,10 +81,11 @@ def test_redis_default(aggregator, redis_auth, redis_instance):
     for name in aggregator.metric_names:
         if name in DB_TAGGED_METRICS:
             aggregator.assert_metric(name, tags=expected_db)
-        elif name != 'redis.key.length':
+        elif name != 'redis.key.length' and name != 'redis.net.connections':
```

Comment:
(Nit) Makes it easier to ignore more in the future:

```suggestion
        elif name not in ('redis.key.length', 'redis.net.connections'):
```